File: Dominio/ObjetosDeValor/Embarcador/Enumeradores/EnumFormaPagamento.cs
namespace Dominio.ObjetosDeValor.Embarcador.Enumeradores
{
    public enum FormaPagamento
    {
        Nenhum = 0,
        Avista = 1,
        GerarTituloAutomaticamente = 2,
    }

    public static class FormaPagamentoHelper
    {
        public static string ObterDescricao(this FormaPagamento situacao)
        {
            switch (situacao)
            {
                case FormaPagamento.Avista: return "Á vista";
                case FormaPagamento.GerarTituloAutomaticamente: return "Gerar Título Automaticamente";
                default: return "Nenhum";
            }
        }      
    }
}

File: SGT.WebAdmin/ViewsScripts/Enumeradores/EnumFormaPagamento.js
var EnumFormaPagamentoHelper = function () {
    this.Nenhum = 0;
    this.Avista = 1;
    this.GerarTituloAutomaticamente = 2;
};

EnumFormaPagamentoHelper.prototype = {
    obterOpcoes: function (opcaoSelecione) {
        var arrayOpcoes = [
            { text: 'Nenhum', value: this.Nenhum },
            { text: "Á vista", value: this.Avista },
            { text: "Gerar Título Automaticamente", value: this.GerarTituloAutomaticamente },
        ];

        return arrayOpcoes;
    }
};

var EnumFormaPagamento = Object.freeze(new EnumFormaPagamentoHelper());